sdk/monitor/azquery/models.go

```diff
@@ -258,15 +258,14 @@ type MetricValue struct {
 	Total *float64 `json:"total,omitempty"`
 }
 
-// MetricsClientListMetricDefinitionsOptions contains the optional parameters for the MetricsClient.ListMetricDefinitions
-// method.
-type MetricsClientListMetricDefinitionsOptions struct {
+// MetricsClientListDefinitionsOptions contains the optional parameters for the MetricsClient.ListDefinitions method.
+type MetricsClientListDefinitionsOptions struct {
```

Comment:
Have you already considered ~MetricsClient~ListDefinitionsOptions?

Reply:
This is a code generation convention. The naming convention for options is ClientName + MethodName + Options. Don't think I can get it to remove the client name for just this option.